features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/RecordedDataProcessor.kt

```diff
@@ -72,7 +72,6 @@ internal class RecordedDataProcessor(
         val wireframes = snapshots.flatMap { nodeFlattener.flattenNode(it) }
 
         if (wireframes.isEmpty()) {
-            // TODO: RUMM-2397 Add the proper logs here once the sdkLogger will be added
```

Comment:
Just making sure we don't want to log anything here if there are no wireframes.

Reply:
yes we do not as it might happen and it is normal...and also there is no actionable thing I can think of right now